lib/edoc/src/edoc.app.src

```diff
@@ -1,26 +1,31 @@
 % This is an -*- erlang -*- file.
 
 {application, edoc,
- [{description, "EDoc"},
+ [
+  {description, "EDoc OTP library"},
   {vsn, "%VSN%"},
   {modules, [edoc,
+	     edoc_cli,
 	     edoc_data,
 	     edoc_doclet,
+	     edoc_doclet_chunks,
 	     edoc_extract,
 	     edoc_layout,
+	     edoc_layout_chunks,
 	     edoc_lib,
 	     edoc_macros,
 	     edoc_parser,
 	     edoc_refs,
 	     edoc_report,
 	     edoc_run,
 	     edoc_scanner,
-             edoc_specs,
+	     edoc_specs,
 	     edoc_tags,
 	     edoc_types,
 	     edoc_wiki]},
   {registered,[]},
-  {applications, [compiler,kernel,stdlib,syntax_tools]},
+  {applications, [compiler, kernel, stdlib, syntax_tools]},
   {env, []},
-  {runtime_dependencies, ["xmerl-1.3.7","syntax_tools-1.6.14","stdlib-2.5",
-			  "kernel-3.0","erts-6.0"]}]}.
+  {runtime_dependencies, ["xmerl-1.3.7", "syntax_tools-1.6.14", "stdlib-2.5",
```

Comment:
```suggestion
  {runtime_dependencies, ["xmerl-1.3.7", "syntax_tools-1.6.14", "stdlib-@OTP-17120@",
```

Just realized that since we use functionality from `shell_docs` we need to update this.